test/integration/service_sign_in/choose_sign_in_test.rb

```diff
@@ -55,9 +55,9 @@ class ChooseSignInTest < ActionDispatch::IntegrationTest
 
       click_on 'Continue'
 
-      assert page.has_css?(".gem-c-error-summary")
-      assert page.has_css?(".gem-c-error-summary__title", text: 'You haven’t selected an option')
-      assert page.has_css?(".gem-c-error-summary__link[href='#option-0']", text: 'Please select an option')
+      assert page.has_css?(".govuk-error-summary")
```

Comment:
These assertions were too tightly coupled to the component implementation in the gem. To avoid this happening again, perhaps we can just test for the existence of "You haven’t selected an option" on this page, without checking for the CSS generated by the gem (which should be internal).